src/data_provider/src/packages/packagesLinuxParserHelper.h

```diff
@@ -56,7 +56,7 @@ namespace PackageLinuxHelper
                 }
 
                 ret["name"]         = name;
-                ret["size"]         = size.empty() || size.compare(DEFAULT_VALUE) == 0 ? 0 : stoi(size);
+                ret["size"]         = size.empty() || size.compare(DEFAULT_VALUE) == 0 ? 0 : stol(size);
```

Comment:
I think you forgot to update parseDpkg as well.

Reply:
You're right. I'll do it in the next commit